source/utilities.tex

```diff
@@ -872,6 +877,11 @@
 !is_convertible_v<decltype(get<0>(@\exposid{FWD}@(p))), T1> ||
 !is_convertible_v<decltype(get<1>(@\exposid{FWD}@(p))), T2>
 \end{codeblock}
+The constructor is defined as deleted if
+\tcode{reference_constructs_from_temporary_v<first_type, decltype(get<0>(FWD(p)))>}
+is \tcode{true} or
+\tcode{reference_constructs_from_temporary_v<second_type, decltype(get<1>(FWD(p)))>}
+is \tcode{true}.
```

Comment:
```suggestion
\tcode{reference_constructs_from_temporary_v<first_type, decltype(get<0>(@\exposid{FWD}@(p)))>}
is \tcode{true} or
\tcode{reference_constructs_from_temporary_v<second_type, decltype(get<1>(@\exposid{FWD}@(p)))>}
is \tcode{true}.
```

Reply:
http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2021/p2255r2.html#pnum_42 read like this, but you transcribed it to "`a || b` is `true`". Should this be made consistent?

Reply:
Yes, otherwise LaTeX gives me an overfull `\hbox`.